### PR TITLE
add packer-plugin-googlecompute package

### PR DESCRIPTION
**Pull request by kranurag7**
tests are missing because we don't have packer.

### CODE REVIEW

File: packer-plugin-google.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - uses: go/bump
+    with:
+      deps: github.com/hashicorp/go-retryablehttp@v0.7.7
+
+  - uses: go/build
+    with:
+      packages: .
+      output: packer-plugin-google
+      ldflags: -X github.com/hashicorp/packer-plugin-googlecompute/version.Version=${{package.version}}
+
```
Comment (EyeCantCU) [inline packer-plugin-google.yaml:24]: We should add some form of a package test here

**Review by EyeCantCU — CHANGES_REQUESTED** [review]

File: packer-plugin-google.yaml (hunk 1 above)
Comment (mamccorm) [reply packer-plugin-google.yaml:24]: Re: testing. In the absence of a --help or a --version, can you share info on how we were able to manually test this? It seems like it requires packer to operate, and I don't believe we have a packer apk today

Comment (kranurag7) [reply packer-plugin-google.yaml:24]: Yes that's right. Given packer is BUSL we don't have packer in here but plugins are MPL so plugins can be added to wolfi but open to thoughts. Plugins can be migrated to other repo too.

Comment (mamccorm) [reply packer-plugin-google.yaml:24]: Have we been able to do any manual validation that the plugins are detected and usable by packer?

**Comment by kranurag7** [comment]
closing this for now given this requires more work and we don't have confirmation yet on the tag.

**Pull request closed without merging**